src/algorithm/distance/DiscreteFrechetDistance.cpp

```diff
@@ -147,6 +147,8 @@ DiscreteFrechetDistance::compute(
                 }
         }
         ptDist = getFrecheDistance(ca, pSize-1, qSize-1, *lp, *lq);
+	delete lp;
```

Comment:
caused by mixture of tabs and spaces. I have replaced spaces with tabs in the files that use tab  indentation in bac9a7d